### DISCUSSION TITLE
How to update artist image without having to manually reset cache directory

### DISCUSSION THREAD

**Discussion by dnno** [navidrome/navidrome#2464] [Q&A]
I had the following situation: I had an artist image in a folder that I wanted to update. I overwrote it, but it had no effect. Eventually the only thing that helped was deleting the `data/cache/images` folder. Is there a better way to do this? Or is this uncharted country atm?

**Comment by cvaziri1** [comment]
I have encountered this as well.  My situation was different in that the artist never had an artist.* and was sourcing the artist image from spotify.  Adding artist.jpg to the base directory for the artist did not work, nor did adding it to an album folder.  A full re-scan also had no effect, nor did restarting the app.  Deleting the image cache folder was the only way I found for the image to be displayed.

**Comment by SushiByte-beep** [comment]
I dont't want to necro an old question but this is the top result in my search. 

Is this still the only way?

**Reply by deluan** [reply]
Currently it is. In the new scanner (#2709), updating the image file will invalidate the cached one.
